### ISSUE TITLE
Create documentation of database structure

### ISSUE THREAD

**Issue by MartinBenediktBusch** [lexicongovernance/forum-backend#86]
### Overview 

It would be beneficial to create documentation for reference of the database structure in the readme of the forum-backend. However, this is conditional on the database structure not changing significantly over the course of the next month. Furthermore, once the documentation is available it must be up to date all the time. This means that when a database change occurs this change must be applied in the documentation as well.

**Comment by Ivar246** [comment]
do you mean documentation of database model, and if something change on that model, you want to reflect that automatically on readme.md file?

**Comment by MartinBenediktBusch** [comment]
> do you mean documentation of database model, and if something change on that model, you want to reflect that change automatically on readme.md file?

Good point. We want to simply document the database tables that are a result of the database model. However, we want to describe the relationships between the tables as well which implicitly is a description of the database model. We are not thinking to update it automatically to start out with.
